data/src/main/java/com/mashup/alcoholfree/data/di/DataStoreModule.kt

```diff
@@ -0,0 +1,41 @@
+package com.mashup.alcoholfree.data.di
+
+import android.content.Context
+import androidx.datastore.core.DataStore
+import androidx.datastore.core.handlers.ReplaceFileCorruptionHandler
+import androidx.datastore.preferences.SharedPreferencesMigration
+import androidx.datastore.preferences.core.PreferenceDataStoreFactory
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.emptyPreferences
+import androidx.datastore.preferences.preferencesDataStoreFile
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+import javax.inject.Singleton
+
+@InstallIn(SingletonComponent::class)
+@Module
+class DataStoreModule {
+
+    @Singleton
+    @Provides
+    fun providePreferencesDataStore(@ApplicationContext appContext: Context): DataStore<Preferences> {
```

Comment:
```suggestion
    fun providePreferencesDataStore(@ApplicationContext context: Context): DataStore<Preferences> {
```

Nit: 꼭 applicationContext임을 알려주지 않아도 괜찮을거같아. 추가로 명시해주고싶다면 줄여쓰기보단 applicationContext로 풀어서 쓰는게 좋을거같아

Reply:
넵넵 수정하는게 좋을것 같습니다!